encoding/config.go

```diff
@@ -16,22 +16,22 @@
 package encoding
 
 import (
-	"cosmossdk.io/simapp/params"
 	amino "github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/auth/tx"
 
 	enccodec "github.com/evmos/ethermint/encoding/codec"
+	ethermint "github.com/evmos/ethermint/types"
```

Comment:
Ideally we'd shift away from evmos/ethermint. Why are we using evmos/ethermint instead of our own types from types/encoding.go, which also export EncodingConfig?

Reply:
We shouldn't need it

Reply:
Opened https://github.com/zeta-chain/ethermint/issues/85